app-containers/docker/autobuild/overrides/usr/lib/systemd/system/docker.service

```diff
@@ -1,38 +1,46 @@
 [Unit]
```

Comment:
Where did this service come from (you wrote it, or from another distribution)? If the latter, please note the source of the service file in your Git commit message.